packages/samples/headless-react/src/App.tsx

```diff
@@ -96,6 +101,22 @@ const fileTypeFacet = buildFacet(engine, {
   options: {field: 'filetype'},
 });
 
+const fileSizeAutomaticNumericFacet = buildNumericFacet(engine, {
+  options: {field: 'size', facetId: 'size-3', generateAutomaticRanges: true},
+});
+const fileSizeManualNumericFacet = buildNumericFacet(engine, {
+  options: {
+    field: 'size',
+    facetId: 'size-4',
+    generateAutomaticRanges: false,
+    currentValues: [
+      buildNumericRange({start: 0, end: 5e3}),
+      buildNumericRange({start: 5e3, end: 50e3}),
+      buildNumericRange({start: 50e3, end: 5e6}),
```

Comment:
Is using exponents really necessary? 😂  Not that it really matters. The bases could all be 5 here for consistency

Reply:
it's not necessary, I just thought it was more easy to interpret how many KiB, MiB, etc that those represent, since otherwise we'd be counting the zeroes anyway. I could alternatively replace them with their numbers that include the zeroes, or I could create `KiB`, `MiB`, etc constants